config/check_deps.mk

```diff
@@ -0,0 +1,18 @@
+define n
+
+
+endef
```

Comment:
What is that for?

Reply:
It's for spacing and newlines in the printed output of the warning and info messages later in this document. See the references to `$n` there.